README.md

```diff
@@ -6,6 +6,20 @@
 
 ## INSTALLATION
 
+### pip/setup.py
+
+CatMap can be installed directly via pip: 
+
+    pip install --upgrade https://github.com/medford-research/catmap/zipball/master
```

Comment:
Great that pip install instructions are added. It should really point to the upstream, though.

Reply:
Well pointed, Martin. I will amend the pull request. Thanks.